src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -360,6 +361,15 @@
     FLUSH_LOCAL_SCHEMA_CHANGES("cassandra.test.flush_local_schema_changes", "true"),
 
     TOMBSTONE_HISTOGRAM_TTL_ROUND_SECONDS("cassandra.streaminghistogram.roundseconds", "60"),
+
+
+    /**
+     * {@link }}StorageCompatibility mode sets how the node will behave, sstable or messaging versions to use etc according to a yaml setting. 
```

Comment:
Nit: the link tag seem malformed.

Reply:
Done

Reply:
It has been changed to `{@link StorageCompatibility}`, it should be `{@link StorageCompatibilityMode}`

Reply:
Done